src/modules/displayData.js

```diff
@@ -1,33 +1,63 @@
 import { addPopUpEvent } from './popup-details.js';
+import { likesEvent } from './likes.js';
+import fetchLikes from './fetchLikes.js';
+import itemCounter from './itemCounter.js';
+
+const display = (meal) => {
+  const mealContainer = document.createElement('div');
+  mealContainer.className = 'mainItemContainer';
+  mealContainer.innerHTML = `
+    <div class="itemContainer">
+        <a class="itemImageContainer" href=""><img class="itemImage" src="${meal.strMealThumb}" alt="meal picture"></a>
+        <div id="hello" class="itemNameContainer">
+          <h2 class="itemName">${meal.strMeal}</h2>
+            <div  class="likesContainer">
+             <i class="fa-solid fa-heart heart like${meal.idMeal}"></i>
+             <h4 class="likeCounter">${meal.likes} likes</h4>
+            </div>
+        </div>
+        <div class="commentButtonContainer">
+        <button id="${meal.idMeal}" type="button" class="commentsButton">Comments</button>
+      </div>
+      </div>`;
+  return mealContainer;
+};
```

Comment:
The last two Meals in your UI aren't showing, and Upon inspection of the element, the ID, Src, are 'Undefined'. Please take a look at the code logic and fix this.

![null-img-foodHub](https://user-images.githubusercontent.com/107632780/203726931-94308f7f-4670-4cbc-bc55-fbc32a0dabcd.png)